catalogs/glade-na-cordex.json

```diff
@@ -0,0 +1,41 @@
+{
+  "esmcat_version": "0.1.0",
+  "id": "glade-na-cordex",
+  "description": "This is an ESM collection for NA-CORDEX data accessible on NCAR's GLADE disk storage system in /glade/collections/cdg/data/cordex/data",
+  "catalog_file": "path/to/glade-na-cordex.csv.gz",
+  "assets": {
+    "column_name": "path",
+    "format": "netcdf"
+  },
+  "attributes": [
+    { "column_name": "variable", "vocabulary": "" },
+    { "column_name": "scenario", "vocabulary": "" },
+    { "column_name": "driver", "vocabulary": "" },
+    { "column_name": "rcm", "vocabulary": "" },
+    { "column_name": "frequency", "vocabulary": "" },
+    { "column_name": "grid", "vocabulary": "" },
+    { "column_name": "biascorrection", "vocabulary": "" },
+    { "column_name": "common", "vocabulary": "" }
+  ],
+  "aggregation_control": {
+    "variable_column_name": "variable",
+    "groupby_attrs": [
+      "scenario",
+      "driver",
+      "rcm",
+      "frequency",
+      "grid",
+      "biascorrection"
+    ],
+    "aggregations": [
+      {
+        "type:" "union",
```

Comment:
`"type:" "union",` ---> `"type":"union",`

Reply:
You may also want to use a json formatter to make it easy to spot some of these errors: https://jsonformatter.curiousconcept.com/